Firestore/Example/FuzzTests/FuzzingResources/Serializer/Corpus/ConvertTextToBinary.sh

```diff
@@ -1,6 +1,6 @@
-#!/bin/bash
+#!/bin/bash -l
```

Comment:
What's the rationale for adding `-l` to the bash command-line arguments?

Reply:
Nevermind. I forgot to discard this comment and I see that you address this explicitly in the comment added to this file.